### PR TITLE
add manual docker commands to README

### PR DESCRIPTION
**Pull request by chesshacker**
We ran into some issues using docker-compose on Carina, so we found it helpful to also know the manual docker commands to run containers. In case others have to do the same, we thought it would be useful to capture the commands somewhere, and this README seemed like the best place.

### CODE REVIEW

**Comment by chesshacker** [comment]
@smashwilson - This is what @atarola came up with when we were deploying to Carina.  I think it makes sense to capture here... but I wanted to run it by you first.  Thanks!

**Comment by etoews** [comment]
FWIW, docker-compose not working well on Carina is a known issue. It's something that should be resolved when we get overlay networks on Carina next year.

We're planning to port deconst to Carina next year as well. I'm currently looking at some service discovery stuff that will allow us to stitch the services together dynamically.

But whatever happens this is good stuff.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,43 @@ export RACKSPACE_APIKEY=...
...
+
+# build and deploy the content service
+cd {wherever you have the deconst/content-service}
+docker build --tag content-service:1.0.0 .
+docker run -d -p 9000:8080 \
+              -e NODE_ENV=development \
+              -e STORAGE=memory \
+              -e MONGODB_URL=mongodb://mongo:27017/content \
+              -e ELASTICSEARCH_HOST=http://elasticsearch:9200/ \
+              -e ADMIN_APIKEY=$RACKSPACE_APIKEY \
```
Comment (smashwilson) [inline README.md:46]: This isn't used as a Rackspace credential - it's an API key that the content service will _recognize_, not one that it _authenticates with_. I usually generate a random string using something like:

``` bash
hexdump -v -e '1/1 "%.2x"' -n 128 /dev/random
```

File: README.md [hunk 2]
```diff
@@ -28,6 +28,43 @@ export RACKSPACE_APIKEY=...
...
+              --link mongo:mongo \
+              --link elasticsearch:elasticsearch \
+              --name content \
+              content-service:1.0.0 script/inside/dev
```
Comment (smashwilson) [inline README.md:50]: You don't actually need to build the Docker images from source unless you've made changes locally; you can use the automated builds from quay, instead, and skip cloning any repos or running any `docker build` commands.

``` bash
docker run -d -p 9000:8080 \
  # ... \
  quay.io/deconst/content-service

docker run -d -p 80:8080 \
  # ... \
  quay.io/deconst/presenter
```

Comment (smashwilson) [inline README.md:50]: Oh wait, you're setting this up to be a development environment? Oh, that makes more sense.

**Comment by smashwilson** [comment]
Hmm, do you think this might belong in the README for [deconst/integrated](https://github.com/deconst/integrated) instead? That's where I've been keeping the most up-to-date docker-compose setup for local integration testing. I, uh, actually completely forgot that we had a compose file here and instructions in the README.

I could also see it belonging (additionally) in the [deconst docs themselves](https://github.com/deconst/deconst-docs/blob/master/developing/setup.rst) so it'll show up on [deconst.horse](https://deconst.horse/developing/setup/).

**Comment by chesshacker** [comment]
@smashwilson All your suggestions sound good to me.  I will update accordingly and send a PR to both the repos you mentioned.

**Comment by chesshacker** [comment]
@smashwilson - I closed this issue and created two new PRs:
https://github.com/deconst/deconst-docs/pull/213
https://github.com/deconst/integrated/pull/9

**Pull request closed without merging**